src/php/php_aliases.go

```diff
@@ -0,0 +1,123 @@
+package php
+
+var Aliases = map[string]string{
```

Comment:
In `src/ir/phpcore/phpcore.go` it has this. Let's summarize this...

Reply:
Fixed